website/pages/docs/k8s/crds.mdx

```diff
@@ -0,0 +1,482 @@
+    subset1:
+      filter: filter1
+      onlyPassing: true
+    subset2:
+      filter: filter2
+  redirect:
+    service: redirect
+    serviceSubset: redirect_subset
+    namespace: redirect_namespace
+    datacenter: redirect_datacenter
+  failover:
+    failover1:
+      service: failover1
+      serviceSubset: failover_subset1
+      namespaces: failover_namespace1
+      datacenters:
+        - failover1_dc1
+        - failover1_dc2
+    failover2:
+      service: failover2
+      serviceSubset: failover_subset2
+      namespaces: failover_namespace2
+      datacenters:
+        - failover2_dc1
+        - failover2_dc2
+  connectTimeout: 1000000000
+  loadBalancer:
+    policy: policy
+    ringHashConfig:
+      minimumRingSize: 1
+      maximumRingSize: 2
+    leastRequestConfig:
+      choiceCount: 1
+    hashPolicies:
+      - field: field
+        fieldValue: value
+        cookieConfig:
+          session: true
+          ttl: 1
+          path: path
+        sourceIP: true
+        terminal: true
+```
+
+Notes:
+
+1. The name of the service that this `ServiceResolver` configures is set by
+   `metadata.name`.
+1. See [service-resolver](/docs/agent/config-entries/service-resolver) for full
+   documentation on this config entry.
+
+### ServiceRouter
+
+[service-router](/docs/agent/config-entries/service-router)
+configures traffic routing and manipulation at networking layer 7 (e.g. HTTP).
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceRouter
+metadata:
+  name: your-service-name
+spec:
+  routes:
+    - match:
+        http:
+          pathExact: pathExact
+          pathPrefix: pathPrefix
+          pathRegex: pathRegex
+          header:
+            - name: name
+              present: true
+              exact: exact
+              prefix: prefix
+              suffix: suffix
+              regex: regex
+              invert: true
+          queryParam:
+            - name: name
+              present: true
+              exact: exact
+              regex: regex
+          methods:
+            - method1
+            - method2
+      destination:
+        service: service
+        serviceSubset: serviceSubset
+        namespace: namespace
+        prefixRewrite: prefixRewrite
+        requestTimeout: 1000000000
+        numRetries: 1
+        retryOnConnectFailure: true
+        retryOnStatusCodes:
+          - 500
+          - 400
+```
+
+Notes:
+
+1. The name of the service that this `ServiceRouter` configures is set by
+   `metadata.name`.
+1. See [service-router](/docs/agent/config-entries/service-router) for full
+   documentation on this config entry.
+
+### ServiceSplitter
+
+[service-splitter](/docs/agent/config-entries/service-splitter)
+configures splitting incoming requests across different subsets of a single service.
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceSplitter
+metadata:
+  name: your-service-name
+spec:
+  splits:
+    - weight: 50.1
+      service: foo
```

Comment:
Seeing no difference between the serviceSubset makes this confusing. This would have no real affect then I'm thinking.